### DISCUSSION TITLE
关于AI和未来

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/ClubHouse2024#19] [General]
Geoffrey Hinton & Fei-Fei Li 
https://youtu.be/E14IsFbAbpI?si=5Umrf1u9mRkpbpNZ

openAI devday 
https://youtu.be/lydmOkYSLyE?si=-RjQjloKRxAq4hvH

**Comment by Lewisrxliu** [comment]
https://openai.com/blog/openai-announces-leadership-transition

**Reply by Lewisrxliu** [reply]
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/128250203/9aae67cf-8557-4813-86d7-1a9788492492)

**Reply by Lewisrxliu** [reply]
戏剧性的变化，背后应该有很强的逻辑
https://twitter.com/OpenAI/status/1727206187077370115?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Etweet
https://twitter.com/sama/status/1727207458324848883?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Etweet
https://twitter.com/ilyasut/status/1726590052392956028?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Etweet

**Reply by too** [reply]
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/171673/918ee665-e161-4884-be55-909d2538a9df)

**Reply by guttasingapore** [reply]
https://36kr.com/p/2528961938335233

**Comment by Lewisrxliu** [comment]
Ilya speech (2023 Oct), TED chose the best timing release this episode 
https://youtu.be/SEkGLj0bwAU?si=8PXtVjAB_qqYbV_x

**Reply by Lewisrxliu** [reply]
自我反省，8个月的学习，学到了些啥？说不上来，越来越觉得自己无知
Referring The Dunning-Kruger Effect, I'm still in the valley of stupidity, porosity and curiosity.
https://bbs.deepin.org/phone/en/post/253881

**Comment by ergatea** [comment]
三月就用python调用API了，厉害！


lewis ***@***.***> 于2023年11月21日周二 20:38写道：

**Reply by Lewisrxliu** [reply]
我很惭愧，因为这个被女儿批评得无地自容。厉害的当然是年轻人，看看这个小伙子, 听说就是一杯咖啡的时间 Edward Tian & his GPTZero
https://gptzero.me
https://www.linkedin.com/in/ed-tian
https://twitter.com/edward_the6?lang=en
https://m.youtube.com/watch?v=-6E2cgEiMeU

**Comment by Lewisrxliu** [comment]
Today learning from Google Search central live (22 Nov 2023 @ Google space Blk 70 Singapore): #SEO # organic search !
[image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/128250203/6b86123d-e980-4198-8ae3-fc2463c5b007)

Gary
1) Crawling, indexing, search & ranking 
2) content & search console: web pages, images, news articles, local, Video
3) status dashboard, News & changes, AI&AGI, SGE, Bard, in coming SearchLab, 
4) Google guideline on AI generated content: reward high quality content. ?
5) deprecate all, data exports

Aaseesh Marina- analysing traffic drop in SC 
1) content creator - site owner - marketer designer editor - SEO engineer & developers
2) performance report: data review & analysing 

My notes: 
1) skill wanted - what & how to - managing infinite/unlimited information by limited resources, 
2) my creative idea & Phython self study project: AI assist Long Video shortening to clip ready for Video group marketing? Talk to Cassie
3) my assumption: what if the BU most profitable  business facing BIG challenges (here searching for Google), why/what/how?

Event Agenda
How Search Works
What's new in Search
Analysing traffic drop on Search Console
Aiyo! Why you update again ah?
Al and Search
SEO Mythbusting
Partnering with Google Search
SEO for videos
Break
Internationalisation and Localisation
15:50 - 16:00 Rich results for Ecommerce sites
I got large sites. Got recommendations?
Preparing for the end of third-party cookies

Search Central Live Singapore 2023
resources: g.co/searchcentral
questions: g.co/searchcentral/help
Twitter: @googlesearchc
Youtube: Google Search Central

**Comment by Lewisrxliu** [comment]
Interesting view
https://www.businesstimes.com.sg/singapore/economy-policy/small-business-owners-warm-ai-written-contracts-lawyers-warn-costly

**Comment by Lewisrxliu** [comment]
Thanks Dhi for this typical story, worth to drill and learn more

1)Vesuvius Challenge is not new, challenges are still alive
https://scrollprize.org
https://youtu.be/QBc4oF7QU2o?si=gLuTeGtC_ljJ19e1
https://zh-cn.futuroprossimo.it/2019/10/i-papiri-di-ercolano-carbonizzati-dal-vesuvio-saranno-letti-dallai/amp/
https://www.311project.com/?post_type=post&p=239695

2)Luke Farritor's achievement - First Letters Prize
https://www.youtube.com/live/kFRIuCMCMUg?si=p600VtFA3uTIgVgQ
https://youtu.be/TC9p6LmrYBo?si=IyxFZ4WkrDQ7uelm
https://m.36kr.com/p/2477212368640132
https://time.com/6326563/vesuvius-challenge-herculaneum-papyri-ai/
https://cloud.kepuchina.cn/h5/detail?id=7113245700608696320

3)AI assisting archaeology is not new, still in fast developing, fascinating 
https://www.bbc.com/news/world-middle-east-56842712.amp
https://toppandigital.com/translation-blog/using-ai-crack-ancient-languages/?amp=1
https://www.lifewire.com/how-ai-translation-is-helping-scientists-understand-ancient-languages-7107026#:~:text=A%20team%20of%20scientists%20at,fragments%20to%20make%20them%20understandable.

Reference
https://www.nature.com/articles/d41586-023-03212-1

https://scrollprize.org/firstletters

https://people.com/21-year-old-wins-usd40k-after-using-ai-to-read-first-word-on-2-000-year-old-papyrus-scroll-8358107

**Comment by Lewisrxliu** [comment]
A debate - what AI revolution means for future: 
from AGI toward ACI, painting the pictures of 2028
https://youtu.be/7JkPWHr7sTY?si=Y9CRH7L6NLEECbEd
By economist 2023.9.11

**Comment by Lewisrxliu** [comment]
> Geoffrey Hinton & Fei-Fei Li 
> https://youtu.be/E14IsFbAbpI?si=5Umrf1u9mRkpbpNZ
> 
Geoffrey Hinton 60 minutes (2023 Oct)
https://youtu.be/qrvK_KuIeJk?si=_sb3-LyYGcXf-xkm

> openAI devday 
> https://youtu.be/lydmOkYSLyE?si=-RjQjloKRxAq4hvH

**Comment by Lewisrxliu** [comment]
LazadaOne现场, 同学们真的都好努力呀！
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/128250203/d8e4f813-ee34-40ca-bbaa-57ebf4fc8781)

**Reply by Lewisrxliu** [reply]
AI+ or +AI?
B2B market behaviour Big Diff between CN & Oversea
， 传统 How to competing against BIG Bro?
Now：AI 带动生产力的进步
BLC 带动生产关系的进步
私链萌链 metaverse 都是耍流氓 游戏升级版？
不赚钱的是不是不可持续的，需要小而美的vertical
所有的Web3现在都是骗子吗？
solana investor
future：必须依靠年轻人 genZ, BLC 因应年轻人的需求
未来的可能：跨境快捷支付，游戏虚拟世界虚拟资产， 平行世界和社交需求
其他：传统金融与商业模式重建， 虚拟货币， 金融自由和法币贬值

**Comment by Lewisrxliu** [comment]
对投研GPT的尝试和思考 (转帖，三天后自动删除）

Original 拉菲  瑞泽洞察  2023-12-02 11:16

https://mp.weixin.qq.com/s/EMD-5AFngSab4ooOLSwLUA

迈入2023年最后一个月，如果要概括中国AIGC的当前现状，大概是这样：论坛大会云集、项目乏善可陈、论调大同小异。上周末在北京参加了三场活动：奇绩创坛秋季Demo Day, 牛文文的黑马企业家论坛，以及虎嗅FM节。三个风格迥异的科技IP，请来的主角都是AI。奇绩的活动人满为患，说是汇聚了中国最优秀AI年轻创业者和投资人，也不为过，毕竟有陆奇老师这样的顶流大咖操持。问了几个同去参加Demo Day的投资人，觉得项目团队普遍挺棒，但能出手的项目寥寥无几。说是800多个项目里挑出60个，但能看一看的可能就十来个。大部分是应用层项目，2B/2C大概各一半，每个项目几分钟的路演中，最大感受是偏理想主义，对市场的理解也不够深。有不少是工作3-5年甚至在校生，缺乏商业意识和人脉也是个挑战。对比了另外两场和这几天陆续进行的，嘉宾也是翻来覆去老面孔，百川王小川、智源黄铁军、智谱张鹏、红衣教主周鸿祎，见不到的始终见不到，比如Minimax闫俊杰。这非常中国风，明明国内已有百个大模型，看似热闹，唱戏的就那么几家。难怪小川总说，国内大模型最终就三家，算上百川和文心一言，那剩下一张门票是谁的？字节？阿里？Minimax？智谱？智源？Moonshot？每家都不会说自己差，一年后继续看了。抛开国内大模型圈子，国内2B应用层的发展还处于超头部甲方的浅尝阶段，确实有不少大甲方尤其是金融、能源体系的增加了明年相应的IT预算。很显然，AIGC应用的初始商业化方向，又得从央国企大甲方开始。这是机会，初创企业进入央国企供应商名单也只有这种技术红利下才有可能，优秀的项目团队应该快速证明自己。但为什么要说乏善可陈？核心原因是众所周知的，哪怕基于GPT4的复杂推理要求，效果就明显折扣。毕竟大部分2B场景对精准度、专业度要求极高，20%的水准下降，或3-5%的错误率都是不被允许的。我们也在自己的小领域做了泛投研GPT问答产品的开发。首先，GPT4下多个方法跑出来的实测结果，明显优于国内大模型，团队背景是CMU算法方向本硕博水准，且有十年以上工作经验，这很重要。但即便如此，当我们把常规行研框架下的二三十个问题分成不同类型和难度，总有那么20%的问题始终跑不通。这期间，我们也体验了几家同行做的泛投研类GPT，同时问了一圈身边用过的投资人，均不尽如人意。前阵子云栖大会上，也和通义千问下金融场景的算法人员交流了下，只能回答最简单的、基于具体数字、名称的一些事实结果，稍微难一点的问题就暴露了。比如，当我们在问市场阻碍因素、产品发展趋势时，基于国内模型清一色是车轱辘话。另一侧，基于GPT4下，跑了1-2周，尝试不同prompt方法后，可以不断改善结果，但数据源必须足够优质，且一定是基于小样本。一旦多喂几份报告或访谈纪要，哪怕每份质量很高，回答质量反而下降。再有，起初问到产品趋势、技术趋势、市场趋势时，GPT4给的反馈几乎是不分的，这其中就需要真的挺懂投资研究的人，反复和算法工程师沟通，他明白了，再用机器能明白的方法，打造基于这套逻辑的训练集群。我们也尝试问些高级问题，如行业的终局会是什么样的？回答的不确定性更高，但内容质量和人肉输出相比，暂时无法相提并论。其他场景或基于多模态输出的产品，结果会好很多，比如今天被炒的Pika这个场景，AI视频剪辑我们在2年多前就做过几个项目的商业尽调，里面一些原理和生成结果还是比较落后的。基于GPT多模态能力，生成效率和质量是传统AI模型望尘莫及的。这些项目特征很符合美元投资人胃口，上述提及的投研类GPT，有的早期项目上来就几亿估值，方法和效果大家都差不多(门槛并不高)，也是挺震惊。毕竟，北森上市前~30亿美金估值，如今~3亿美金市值。回到央企甲方那par，所以怎么真正在国内AIGC大浪潮中赚到钱？先得看清楚大方向、大方针，以免重复出现创投圈的北森困境，投资人没赚钱，创始人还得倒吐。那大方向是什么？即在党的全面领导下，算力基建由华为和一批半导体国家队主导；数据由国家数据局主导，数据资产入表已经开始；算法不管谁胜出，都会有国资身影；内容由宣传部把控大原则、出意见。以上不仅正在发生，也在加强中，跟从就好。未来20年最大的阿尔法是政策，因为很多人还没看明白或无法认同。大方向清楚了，还得缕清技术、产品、运营、商业资源的关系。真想这波占领先机的创业者，要深刻领悟上一轮国内SaaS狗血剧情背后的点点滴滴和系统成因。可以翻下瑞泽老文，如聊了800多位SaaS专家和50多位创始人后，总结出的SaaS投资思考，如AIGC商业化并不乐观。具体到产品层面，还是要见实效的。你的产品在2B场景下，专业人员怎么看？会怎么用？企业怎么看？愿意付多少钱？以及这些问题背后的核心逻辑。这些老生常谈了。再具体到产品侧的数据问题，比如泛投研场景，未来真正好产品一定要有独家的、高质量的数据源，这是功夫活。市面上泛泛的研究资料不值钱，手握几万份访谈纪要的人也不少，我们也有。这些纪要一方面真假难辨，另一方面，第二年就剩残余价值了。这一领域最根本的能力/价值体现是，怎么在数据里挖到宝？几十万企业级数据库我们也有，谁才是优质水下项目？算法团队当然是重要的，但就AIGC应用层，国内清北本硕的算法背景，加上5-10年大厂产品经验，妥妥够了。最终是看团队配合，怎么在严苛的2B场景下，把数据源做深做细，做出专属价值，怎么打磨自己的工程化方法，在下一个革命性GPT X来临之前，抢占先机。最后是竞争策略，只有在不同规模企业里参与过各类战役的人，才知道怎么应对。这部分的能力经验是举足轻重的，待2-3年后国内top3大模型成熟，又是一个场景之下一堆同质化产品，是你，现在怎么布局？从AIGC到AGI大方向，毫无疑问是未来10年乃至20年仅剩的超级机会，每位创业者和企业都得重视起来，连我们一个小破研究团队都无比重视。最近除了上述投研GPT开发外，也做了一款投研小工具，暂时内部自用，数据质量、数量已和头部产品相当，也做了一些特色。我们把1-2万个硬科技融资项目，人肉投入进行1-4级分类，不重、不漏、投研级标签，而不是市面上大部分产品依赖工商标签，那个简单。因为标签细了，相关竞品分析、相关报告的功能就比同行做的更准一些。肯定会有瑕疵，持续优化中。
光有数据库肯定是不够的，GPT问答功能暂时又有点鸡肋。接下来3-4个月，我们会做几个和友商差异化的刚需功能，尝试解决之前产品没能帮投资人解决的问题。大家都知道一级市场很差，所以创投数据产品的未来买家不是GP，应该是政府招商+GP才对，这也是为什么市场那么差，我们还要做一个数据库。招商数据+标准服务(若干)+落地能力，未来既是招商必须，也是创投必须。上头很明确说了，党管金融,特别是未来面临持续的不确定性时，体制决定了是党的信用来兜底，和华尔街金融政治那套不一样。那现在国内经济最紧迫的是什么？国内300个地级市和近3,000个县级市，卖地收入原是大头，现在锐减到可怜，面临着重大的动能调整，一下子根本转不过来。今年各级政府早就是everybody招商了，GP无非是政府招商工作的外包商。这个市场过去不太需要被服务，坐等开发商来勾地就行，催生了每年十几万亿人民币的生意。未来20年怎么办呢？没人把钱送上门了，能力也非一时半会能提升，连成都公交车公司都发不出工资。一面是各级政府财政收入出现那么大的窟窿待填补，另一面是深圳这些地方的头部制造企业的成本转移需求，以及科创板第五条永久关闭后，一堆靠狂亏到敲钟的创始人，都开始找政府要救命钱了。供需都强烈，目前没有针对性的专业服务机构。至于怎么用数据+服务来助力新一代创投+招商形态，这几个月聊了些业内招商大拿，也是术业有专攻，但方向是对的，先干再说。目前除长三角、珠三角和安徽，各级政府普遍缺乏真正的、可持续的、在市场高度不确定性下的招商能力，这里面有坑、有风险，也有持续的服务机会。目前GP招几个招商经理的方式，估计很难做起来。可是做不好的话，都是签了字的，到时完不成不是开玩笑的。讲回初心。数据产品只是底层基础，这市场很小，头部2家也就几亿估值，但我们看到了AIGC下整个国内研究咨询、FA类服务的变革机会，这一切的基础是搭建私有高价值的数据源，持续跟着Top大模型提升工程化能力，叠加我们现有的优势，比如资深的投研能力、一定的2B流量基础。10年后中国最头部的本土咨询研究公司，肯定不是今天这几家，他们在这个领域面临和阿里一样的挑战。我们会全力投入，尽管目前差距甚大。最后转一个黄峥2016年时的思考，很震惊。那是阿里最辉煌阶段，当时没人相信综合电商怎么可能还有机会？但他说了下面这么一段话，然后用7年时间干了一家2K亿美金的企业，超越了对手。早期的拼好货甚至有点干不下去，但黄峥最终把这些不可能的想法，今天一一全部实现。慢慢来，寒冬之中，更不急一时。

**Comment by Lewisrxliu** [comment]
S'Pore的SCAI
https://www.scai.gov.sg/community/

![image](https://github.com/RoadSideUniversity/ClubHouse2023/assets/128250203/919c63a9-42ae-458a-9a80-8fdd74385724)

SCAI concluded 12 questions
https://www.zaobao.com.sg/news/singapore/story20231207-1454628?amp
https://www.scai.gov.sg/foreword-scai-questions/
https://file.go.gov.sg/scai-questions.pdf

S'Pore 人工智能策略2.0（National AI Strategy 2.0 outline 15 actions
https://www.zaobao.com.sg/news/singapore/story20231204-1454163
https://www.smartnation.gov.sg/nais/
https://file.go.gov.sg/nais2023.pdf

**Reply by guttasingapore** [reply]
https://sbr.com.sg/information-technology/news/spore-surpass-us-in-ai-investments#:~:text=AI%20investment%20by%20the%20Singapore,years%20compared%20to%20other%20nations.

S’pore to surpass the US in AI investments
AI investment by the Singapore government is about 16% more compared to the US.

Singapore registered the highest sum in AI investments against the gross domestic product (GDP), with a relatively modest investment of $9.4b (US$7b) in the last five years compared to other nations.

According to AIRPRM's report with data from OECD and the World Bank, this equates to more than 15.0% of Singapore's relative wealth per thousand dollars of GDP. In 2023 alone, the country invested $2.5b (US$1.9b) in AI.

Following closely behind is Sweden, allocating 14.1% of its GDP towards the development of AI. In contrast, the US invests approximately 12.9% of its GDP in AI infrastructure, while the UK trails behind at 8.3%.

When assessing AI investment on an international scale in comparison to the US, Singapore once again takes the lead. At 116.3%, the rate of AI investment by the Singaporean government surpasses their US counterparts by around 16% in relative terms per billion dollars of GDP.

**Reply by guttasingapore** [reply]
IMDA, AIVF propose framework to advance generative AI development
The parties aim to finalise the Model AI Governance Framework in mid-2024.
The AI Verify Foundation (AIVF) and Infocomm Media Development Authority (IMDA) developed a draft Model AI Governance Framework for Generative AI.
The draft expands on the existing Model Governance Framework that covers Traditional AI, last updated in 2020.
The proposed framework aims to facilitate international conversations amongst policymakers, industry, and the research community to enable trusted development globally.
The framework will draw on practical insights from ongoing evaluation tests conducted in a Generative AI Evaluation Sandbox.

https://sbr.com.sg/information-technology/news/imda-aivf-propose-framework-advance-generative-ai-development
https://aiverifyfoundation.sg/downloads/Proposed_MGF_Gen_AI_2024.pdf
[Proposed_MGF_Gen_AI_2024.pdf](https://github.com/RoadSideUniversity/ClubHouse2024/files/14045990/Proposed_MGF_Gen_AI_2024.pdf)

**Reply by Lewisrxliu** [reply]
斥资打造AI生态系统 三方面壮大人才库
AiVP, MbR, people
https://www.zaobao.com.sg/news/singapore/story20240301-1471573

**Comment by Lewisrxliu** [comment]
NFT进入博物馆，居然这么滴明目张胆。比桥头同心锁强！
![image](https://github.com/RoadSideUniversity/ClubHouse2023/assets/128250203/fc5be051-087c-4a96-b034-946fab9a1557)
![image](https://github.com/RoadSideUniversity/ClubHouse2023/assets/128250203/c236c076-91c5-4ad9-adae-bd6f8c3821c8)
![image](https://github.com/RoadSideUniversity/ClubHouse2023/assets/128250203/77dc488f-ac5f-4b53-8768-5c2e22bc1d9d)
![image](https://github.com/RoadSideUniversity/ClubHouse2023/assets/128250203/ff757b22-8faf-4f39-ad77-1548c68cdfd7)

**Reply by guttasingapore** [reply]
https://en.wikipedia.org/wiki/The_Kiss_(Klimt)

**Comment by Lewisrxliu** [comment]
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/68b038be-9adc-4846-ae06-5a8dbb3fea1b)

**Comment by Lewisrxliu** [comment]
未来除了避险，还需要积极探索投资方向
FSM invest expo 2024
Congratulation to myself, lucky draw awarded S$100 on 6 Jan 2024
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/b7f8319f-1a8b-4c56-be84-e6572662f8c9)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/7bb4ecad-d82a-420d-9459-9697ba4a847e)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/b626153d-b6bf-41c1-913a-61e1f1a225b2)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/586d49ad-00f5-423a-a649-9ecf33aecb29)

**Reply by Lewisrxliu** [reply]
2024 outlook - Invest Expo (reference only)

FSM
https://publications.schroders.com/view/719027494/8/

JP Morgan
https://am.jpmorgan.com/hk/en/asset-management/adv/insights/market-insights/year-ahead/
https://am.jpmorgan.com/content/dam/jpm-am-aem/asiapacific/regional/en/insights/market-insights/year-ahead-2024/MI-WP-YearAhead2024.pdf
https://am.jpmorgan.com/sg/en/asset-management/per/insights/market-insights/guide-to-the-markets/

Blackrock
https://www.blackrock.com/sg/en/literature/brochure/q1-2024-implementation-guide-np-retail.pdf

![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/65d20396-e9dc-4281-b712-3fd4f8e88fd7)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/dea9fd3d-c070-4b3c-8c5d-ef401817aa10)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/1817d436-a202-4e99-88cd-495fb1dd102b)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/77ce8fc8-ffe1-4608-81b0-5479637a5519)

Global inflation
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/39aa0b6c-d379-4d57-9ec3-0698b6ac304b)

**Comment by Lewisrxliu** [comment]
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/6446fb31-febc-477d-abbf-530fbdec8e24)

**Comment by Lewisrxliu** [comment]
复习 whisper transcribe

https://mbrochh.github.io/whisper-youtube-transcribe/1

**Comment by Lewisrxliu** [comment]
2024.1.9-12 Las Vegas CES
https://www.ces.tech
https://live.ces.tech
网易直播ZhouYu/TCL，1.12 笔记： 供应链创新， CMNC国际化进一步（此次CMNC1400家） VS 疫情前中国厂商曾参展5000家

**Comment by Lewisrxliu** [comment]
无所适从？

https://news.sina.cn/gn/2024-01-17/detail-inacuwni3441592.d.html
当地时间1月16日，中国国务院总理李强出席世界经济论坛2024年年会时表示，人类对人工智能又爱又怕，感觉AI无所不能但感到无所适从。

**Comment by Lewisrxliu** [comment]
AI forward - 阿里云智能大数据峰会
AI Forward: Alibaba Cloud AI & Big Data Summit - Tuesday, 30 January 2024 at 4th Floor, Sands Expo & Convention Centre

https://www.alibabacloud.com/events/ai-big-data-summit-2024
https://m.youtube.com/watch?v=YarL0YsQq1Y
https://twitter.com/alibaba_cloud/status/1752271382401143062?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Etweet

https://www.businesstimes.com.sg/companies-markets/alibaba-exit-cloud-business-after-beijing-undercuts-potential
我在现场
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/4bd816dc-9f63-43a9-9535-fb743337bb50)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/c0045772-d498-4fdb-aa9b-01ec8ed02fed)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/6385a5bf-ca24-47cf-9d79-2d4f154ae5ac)

**Comment by Lewisrxliu** [comment]
努力学习中 Blockchain Analytics on the Cloud 🔍  ☁️ 
Scheduled: 31 Jan 2024 at 19:00 to 21:00, SGT
Location: Google Developers Space, Singapore (80 Pasir Panjang Rd, Singapore, Singapore 117372)
https://www.meetup.com/practical-blockchain/events/298446756/
Practical Blockchain
Wednesday, January 31 at 7:00 PM

Are you curious about blockchain analytics and harnessing the power of the cloud for working with blockchain data? How does blockchain really work, ho...

https://www.meetup.com/practical-blockchain/events/298446756/

**Reply by Lewisrxliu** [reply]
印度小女生Akshata Mohanty探索的区块链溯源，以及她独立创建的区块链应用与云社区
Finding the Genesis Block: My first forays into working with blockchain data
Date: January 30, 2024
https://practicalblockchain.org/2024/blockchain-analytics/01-where-are-the-whales/
https://practicalblockchain.org
https://gdg.community.dev/gdg-cloud-singapore/

需要继续，年轻人的探索精神值得学习
Node Providers & Ethereum clients (web, ethers) Node Providers - Infura, Cloudflare, Quicknode .. 
https://ethereum.org/developers/docs/apis/json-rpc
https://github.com/ethereum/execution-apis
https://docs.ethers.org/v5/
https://okg.go.dev/github.com/ethereum/go-ethereum/ethclient
> node 01-building-a-tracker/currentWalletBalance.js

Challenges
- It's slow to get this data; 10 minutes for 1000 wallet addresses
- Not all of the JSON-RPC schema is implemented by every provider
- Rate-limits, the number of txns coming in, scale management
https://medium.com/google-cloud/ real-time-ethereum-no tifications-for-everyone-for-free-a76e72e45026
https://medium.com/google-cloud/how-to-query-balances
-for-all-ethereum-addresses-in-bigquery-fb594e4034a7

But where did the ETH go?
Get Transaction History
Unfortunately, transaction history is not something that is easy to get. It is not indexed by the blockchain, not by a standard node, "out-of-the-box". At the time of this recipe, the indices to store the entire history are around 800GB. For Parity you may enable tracing and disable pruning, in which case you can use some of the vendor specific JSON-RPC debug methods.
For many cases, you can probably rely on Etherscan, which dedicates large amounts of resources to tracking and storing this information.

How does an indexer work?
Node-Client-ETL Infra-Database
Ethereum in BigQuery: how we built this dataset | Google Cloud Blog
https://github.com/blockchain-etl
Blockchain Analytics 101: Building an Indexer from Scratch by Benjamin
Memisevic / Devcon Bogotá
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/64053d80-f7d0-4812-ad36-abdb5316bc69)

**Comment by guttasingapore** [comment]
努力学AI新知，每天不一样
1.法律法规，
https://www.imda.gov.sg/
2.策略生态，
https://aiverifyfoundation.sg/ai-verify-foundation/
完成今天作业 Completed assessment of [Personal Data Protection Act (PDPA)] (https://www.pdpc.gov.sg/Overview-of-PDPA/The-Legislation/Personal-Data-Protection-Act) 
https://www.pdpc.gov.sg/
https://www.pdpc.gov.sg/help-and-resources/2018/01/e-learning-on-data-protection

**Comment by Lewisrxliu** [comment]
NVIDIA "Chat with RTX" AI Chatbot Is Changing the Game in AI!
https://youtu.be/UKURI5JSIts?si=ymVXBD7KuKUUFXao

**Comment by Lewisrxliu** [comment]
SORA
https://openai.com/sora

**Comment by Lewisrxliu** [comment]
Cyberwar using AI initiated
https://fortune.com/2024/02/14/microsoft-iran-north-korea-russia-china-beginning-use-generative-ai-offensive-cyberattacks/amp/

**Comment by Lewisrxliu** [comment]
> SORA
> https://openai.com/sora

简中解读
https://yunyinghui.feishu.cn/wiki/BaCEwe3AliqYERkc9dVcfW0BnXg?from=from_copylink

**Comment by Lewisrxliu** [comment]
> > SORA
> > https://openai.com/sora
> 
> 简中解读
> https://yunyinghui.feishu.cn/wiki/BaCEwe3AliqYERkc9dVcfW0BnXg?from=from_copylink

技术流推理 by Sora·创世纪·大统一模型
https://mp.weixin.qq.com/s/0JtbSQAQjOs-6sBOydZolA

**Comment by Lewisrxliu** [comment]
Future Frontiers
Navigating the US-China Struggle for Technological Influence in Asia and the World on 18 March 2024, Monday
7:00 PM - 8:15 PM SGT
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/53ca0fd3-8665-4679-967f-3f01be1f5060)
Technological competition between the United States (US) and China is multifaceted, encompassing issues of security, economics, and politics. In the face of this competition, countries and businesses across Asia and around the world strive to manage risk. This panel of experts from the government sector, private industries, and academia examines issues such as US-Chinese discussions on Artificial Intelligence in the wake of the Xi-Biden Woodside Summit; managing competition and finding areas of cooperation; technology and security concerns in the Indo-Pacific region; and how companies are seeking to build resilience in semi-conductor global supply chains and navigate geopolitical risks writ large.
Mr Andrew Makridis
Yale Jackson School of Global Affairs
Assoc Prof Li Chen Renmin University of China
Mr Ted Wittenstein
Yale Jackson School of Global Affairs
Asst Prof Wu Di Renmin University of China
Mr Ziad Haider
McKinsey & Co

**Comment by Lewisrxliu** [comment]
一位商学院教授的公开课
WEALTH MANAGEMENT IN THE AI AGE
AI时代如何管理你的财富？
财富管理1.0到财富管理3.0
03/17.20:00-272-00
北京/新加坡时间（GMT+8）
② 主讲人>>>>
Ben Charoenwong


![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/2e4f3892-873c-49e3-9fa7-dedf5e583e9b)

**Comment by Lewisrxliu** [comment]
Devin：世界首个完全自主的AI软件工程师

![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/c72fbba4-15e6-4e3b-ac3e-61172ca2f029)
https://siliconangle.com/2024/03/12/cognition-launches-devin-generative-ai-powered-coding-engineer/
https://community.openai.com/t/fully-autonomous-ai-software-engineer-devin/679804

**Comment by Lewisrxliu** [comment]
Figure AI, Brett Adcock's Figure 01人形机器人
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/7c316826-c751-4ffc-b837-a1fcfaae1416)
Figure AI 视频中展示的所有行为都是通过学习获得的（非遥控操作），并且以实际速度（1.0 倍速）执行。
Figure AI 将机器人摄像头拍摄的图像和通过板载麦克风记录的语音转录文本输入到一个由 OpenAI 训练的多模态模型中，这个模型能够同时理解图像和文本信息。该模型会处理整个对话的历史记录，包括以往的图像，以生成语言响应，并通过文本到语音的方式向人类回话。同一个模型还负责决定执行哪种已学习的闭环行为来响应给定的命令，它将特定的神经网络权重加载到 GPU 上，并执行相应的策略。
OpenAI 与 Figure 还宣布将共同开发下一代人形机器人 AI 模型，OpenAI 的多模态模型将扩展到机器人感知、推理和交互环节。
Figure 宣布完成了 6.75 亿美元的 B 轮融资，公司估值飙升至 26 亿美元。投资方涵盖了硅谷的半壁江山——Microsoft、OpenAI 创业基金、英伟达、Jeff Bezos、Parkway Venture Capital、Intel Capital 和 Align Ventures 等.

https://m.youtube.com/watch?v=GiKvPJSOUmE
https://www.linkedin.com/posts/brettadcock_openai-figure-conversations-with-humans-activity-7173681973100498944-qVOh
https://en.m.wikipedia.org/wiki/Brett_Adcock

https://www.cnbc.com/amp/2024/02/29/robot-startup-figure-valued-at-2point6-billion-by-bezos-amazon-nvidia.html

**Comment by Lewisrxliu** [comment]
Open Release of Grok-1 on March 17, 2024
A 314 billion parameter Mixture-of-Experts open source model, Grok-1.
https://x.ai/blog/grok-os
https://github.com/xai-org/grok
People
https://x.ai/about
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/1222c94e-6841-468d-a128-f49cf9c5f00c)
